src/modules/integrations/mkcert.nix

```diff
@@ -15,6 +24,26 @@ in
         "*.example.com"
       ];
     };
+    certificateOptions = lib.mkOption {
```

Comment:
let's expose `certFile` and `keyFile` at the top-level without making the implementation about cli options in the options list.

Reply:
Done